data/mods/Xedra_Evolved/spells/classless_spells.json

```diff
@@ -70,5 +70,43 @@
     "min_duration": 21000,
     "max_duration": 90000,
     "duration_increment": 4500
+  },
+  {
+    "id": "damage_transfer",
+    "type": "SPELL",
+    "name": "Damage Transfer",
+    "description": "Allow you to move some damage from your body to someone else.  Usually people don't like it, but who cares now?",
+    "//": "deal damage, give heal. nothing special.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "CONCENTRATE", "SOMATIC", "SILENT", "NO_PROJECTILE" ],
+    "effect": "attack",
+    "shape": "blast",
+    "damage_type": "bash",
+    "min_damage": 5,
+    "max_damage": 25,
+    "damage_increment": 1,
+    "min_range": 1,
+    "max_range": 1,
+    "max_level": 20,
+    "difficulty": 6,
+    "spell_class": "?????????????",
```

Comment:
```suggestion
```
wow that was a huge mistake from my side

Reply:
Ha I actually looked at that and was going to add Classless but apparently forgot to.

Reply:
classless is a separate class? i thought its lack of class, isnt it?